components/media_player/ui/CardTVChannelList

```diff
@@ -0,0 +1,253 @@
+/******************************************************************************
+ *
+ * Copyright (C) 2018-2019 Marton Borzak <hello@martonborzak.com>
```

Comment:
Not me :)

Reply:
Tha's strange I see there my name......nevertheless I pushed it again